run_build.sh

```diff
@@ -2,9 +2,42 @@
 
 export BUILD_DIR="<path_build_area>" #e.g./scratch/POC/BUILDS/
 export IOR_DIR="<path_to_ior_repo>" #e.g./scratch/POC/ior-hpc
-export MPI_DIR="<path_to_mpi>" #e.g./scratch/POC/mpi
 
+# Unload modules that are not needed on Frontera
```

Comment:
move the MPI config back here and source build_env ?

Reply:
Done